airflow/executors/local_executor.py

```diff
@@ -128,28 +133,30 @@ def run(self):
 
 class LocalExecutor(BaseExecutor):
     """
-    LocalExecutor executes tasks locally in parallel. It uses the
-    multiprocessing Python library and queues to parallelize the execution
+    LocalExecutor executes tasks locally in parallel.
+
+    It uses the multiprocessing Python library and queues to parallelize the execution
     of tasks.
     """
     def __init__(self, parallelism: int = PARALLELISM):
         super().__init__(parallelism=parallelism)
         self.manager: Optional[SyncManager] = None
-        self.result_queue: Optional['MultiprocessingQueue[Tuple[ExecutorKeyType, str]]'] = None
+        self.result_queue: Optional['MultiprocessingQueue[TaskInstanceState]'] = None
         self.workers: List[QueuedLocalWorker] = []
         self.workers_used: int = 0
         self.workers_active: int = 0
         self.impl: Optional[Union['LocalExecutor.UnlimitedParallelism',
                                   'LocalExecutor.LimitedParallelism']] = None
 
     class UnlimitedParallelism:
-        """Implements LocalExecutor with unlimited parallelism, starting one process
-        per each command to execute."""
+        """
+        Implements LocalExecutor with unlimited parallelism, starting one process
+        per each command to execute.
+        """
```

Comment:
The line below: `:param executor: the executor instance to implement.` should be here.

Reply:
Fixed.